tasks/apt.yml

```diff
@@ -98,6 +98,7 @@
   apt_repository:
     repo: '{{ item.repo }}'
     state: '{{ item.state | default("present") }}'
+    mode: '{{ item.mode | default("420") }}'
```

Comment:
Good idea. Thank you!

Can you maybe change the explicit default of `default("420")` to `default(omit)` so that we directly fall back to the default value of the module instead of defining the default value of the module once more in this role?